regen/feature.pl

```diff
@@ -1175,6 +1175,8 @@ sub __common {
         my $name = shift;
         if (substr($name, 0, 1) eq ":") {
             my $v = substr($name, 1);
+            carp('Feature bundle ":all" is deprecated and should not be used')
```

Comment:
Use
```perl
warnings::warnif(deprecated => "...");
```
so it's conditional on the category

Reply:
I used `carp` because the rest of the module used `croak` for errors. Is there the equivalent for `carp`? Or should I just not bother?